app.json

```diff
@@ -8,8 +8,7 @@
     "test": {
       "addons": ["heroku-postgresql:hobby-dev"],
       "scripts": {
-        "test-setup": "bundle exec rails ember:install",
-        "test": "bundle exec rspec && bundle exec rails ember:test"
+        "test": "bundle exec rspec"
```

Comment:
The whole `test` attributes here are unnecessary now since we don't test using heroku. Forgot to remove it when we decided to use CircleCI.

Reply:
will clean up